pytest_jira.py

```diff
@@ -146,6 +164,14 @@ def pytest_addoption(parser):
     if os.path.exists('jira.cfg'):
         config.read('jira.cfg')
 
+    try:
+        verify = config.getboolean('DEFAULT', 'ssl_verification')
+    except six.moves.configparser.NoOptionError:
+        verify = True
+    except ValueError:
+        console_write('Wrong argument for ssl_verification!')
```

Comment:
this branch can be easily covered by
```
    testdir.makefile(
        '.cfg',
        jira="\n".join([
            '[DEFAULT]',
            'ssl_verification = something',
        ])
    )
```

and it will cover console_write function as side-effect ...